electronic_store/demo/website_theme_apply.xml

```diff
@@ -0,0 +1,29 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<odoo>
+    <function name="_theme_load" model="ir.module.module" context="{'apply_new_theme': True}">
+        <value model="ir.module.module" eval="obj().env['ir.module.module'].search([('name', '=', 'theme_buzzy')]).ids"/>
+        <value model="website" eval="obj().env.ref('website.default_website')"/>
```

Comment:
I see many industries where we use `model="ir.module.module"`, but this seems to make more sense :thinking: